### PR TITLE
Missing key error Issue#7

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,3 +6,4 @@ gem 'aws-sdk'
 gem 'cucumber'
 gem 'rubocop'
 gem 'trollop', '~> 2.1'
+gem 'pry'
```
Comment (jsywulak) [inline Gemfile]: We should take this out.

File: bin/hamburgerstore.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
 
-require 'hamburgerstore'
+require_relative '../lib/hamburgerstore.rb'
```
Comment (jsywulak) [inline bin/hamburgerstore.rb]: Probably don't want to do require relative here. Let's chat about how to get around that.

File: lib/hamburgerstore-exceptions.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (jsywulak) [inline lib/hamburgerstore-exceptions.rb]: On Keystore I put these into the main class so it was clear there were exceptions for that class. I'm not married to the idea but I'd like us to be consistent across the projects.

File: lib/hamburgerstore-exceptions.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+class HamburgerNoItemInTableError < Exception
+end
+
+class HamburgerKeyNotFoundInItemError < Exception
+end
```
Comment (jsywulak) [inline lib/hamburgerstore-exceptions.rb]: No trailing blank line? :rage3:

**Comment by ghost** [comment]
Ive removed the pry requirement and the exceptions file -- moving the exceptions classes into the model file.

**Pull request merged**